méli-mélo/2023-09-menu/campaign-menu.css

```diff
@@ -0,0 +1,151 @@
+/*** Mega Menu ***/
+/* add background and text color for menu */
+#wb-sm.campaign-menu {
+    background: #003c77;
+    color: #fff;
+}
+/* remove table cell and text shadow*/
+#wb-sm.campaign-menu .menu {
+    border-right: 0px !important;
+    display: block;
+    text-shadow: none;
+}
+
+ /* focus within mneu itme background */
+#wb-sm.campaign-menu .menu .active, 
+#wb-sm.campaign-menu .menu .selected, 
+#wb-sm.campaign-menu .menu .wb-navcurr {
```

Comment:
Need a demo for those 2 CSS selector. I wasn't able to test it with the provided working example.

Suggestion: You can may be change a link in the menu for "index.html" which will enable the `.wb-navcurr` plugin. 

For the `.selected`, I don't see a demo neither able to find a reference in the megamenu source code. Should this CSS selector be removed?